gc/base/standard/Scavenger.cpp

```diff
@@ -3036,8 +3036,12 @@ MM_Scavenger::getFreeCache(MM_EnvironmentStandard *env)
 		OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
 		uint64_t duration = omrtime_current_time_millis();
 
+		bool result = false;
 		omrthread_monitor_enter(_freeCacheMonitor);
-		bool result = _scavengeCacheFreeList.resizeCacheEntries(env, 1+_scavengeCacheFreeList.getAllocatedCacheCount(), 0);
+		cache = _scavengeCacheFreeList.popCache(env);
```

Comment:
We should add a comment here to point out why we're reattempting to acquire a cache